src/fixed-footer-layout.tsx

```diff
@@ -27,23 +33,24 @@ const useStyles = createUseStyles((theme) => ({
         backgroundColor: theme.colors.background,
         transition: 'background 0.2s linear, box-shadow 0.2s linear',
     },
-
     elevated: {
         backgroundColor: theme.colors.backgroundContainer,
     },
-
     withoutFooter: {
         display: 'none',
     },
-
     containerSmall: {
-        paddingBottom: ({height}) => height,
+        paddingBottom: ({footerHeight}) => footerHeight,
         backgroundColor: ({containerBgColor}) => containerBgColor || theme.colors.background,
     },
-
     [theme.mq.tabletOrSmaller]: {
+        containerSmall: {
+            paddingBottom: ({footerHeight, windowHeight}) =>
+                footerHeight * 2 < windowHeight ? footerHeight : 0,
+        },
         footer: {
-            position: 'fixed',
+            position: ({footerHeight, windowHeight}) =>
+                footerHeight * 2 < windowHeight ? 'fixed' : 'relative',
             left: 0,
```

Comment:
to me the windowHeight should be at least 3 times bigger than the fixed footer, otherwise the page would be barely usable:

Please, verify in a real device

![image](https://user-images.githubusercontent.com/579417/196109102-1b1e4cfa-3d99-4b32-9023-57a93b2960b9.png)

Reply:
*4 applied.